packages/vite/src/node/config.ts

```diff
@@ -1666,28 +1671,29 @@ export async function loadConfigFromFile(
     return null
   }
 
-  const isESM =
-    typeof process.versions.deno === 'string' || isFilePathESM(resolvedPath)
-
-  try {
-    const bundled = await bundleConfigFile(resolvedPath, isESM)
-    const userConfig = await loadConfigFromBundledFile(
-      resolvedPath,
-      bundled.code,
-      isESM,
+  if (configLoader !== 'bundle' && configLoader !== 'runner') {
```

Comment:
The judgment here may be moved up, and an error is thrown in advance when the conditions are unmet.

Reply:
Moved it to the start of the function